<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <title>Sequana report</title>
    <link rel="stylesheet" href="css/report.css" type="text/css" />
    <script src="js/sorttable.js"></script>


    <script src="js/jquery.js" type="text/javascript"></script>

    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.3.2/jquery.js" 
        type="text/javascript"></script> 


<script src="http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.js"></script>
<script src="js/galleria-1.4.2.min.js"></script>

<style>
     .galleria{ width: 700px; height: 400px; background: #000 }
$('galleria').galleria({
        imageCrop: false,
        dataSource: data,
        transition: 'fadeslide',
        //transition: "pulse",
        transitionSpeed: 500,
        autoplay: 7000,
        lightbox: true,
    });
</style>






<style>
.slidingDiv {
    height:400px;
    background-color: #99CCFF;
    padding:20px;
    margin-top:10px;
    border-bottom:5px solid #3399FF;
    overflow: scroll;
}
.slidingDivC {
    height:400px;
    background-color: #99CCFF;
    padding:20px;
    margin-top:10px;
    border-bottom:5px solid #3399FF;
    overflow: scroll;
}

.slidingDivStats {
    padding:20px;
    margin-top:10px;
    overflow: auto;
}

.show_hide {
    display:none;
}

</style>
<script type="text/javascript">

$(document).ready(function(){
        $(".slidingDiv").hide();
        $(".show_hide").show();
    $('.show_hide').click(function(){
    $(".slidingDiv").slideToggle();
    });
        $(".slidingDivC").hide();
        $(".show_hideC").show();
    $('.show_hideC').click(function(){
    $(".slidingDivC").slideToggle();
    });
        $(".slidingDivStats").hide();
        $(".show_hideStats").show();
    $('.show_hideStats').click(function(){
    $(".slidingDivStats").slideToggle();
    });
        $(".slidingDivDeps").hide();
        $(".show_hideDeps").show();
    $('.show_hideDeps').click(function(){
    $(".slidingDivDeps").slideToggle();
    });

});

</script>
</head>
<body>

<div class="document" id="unset">
<header>
<h1 class="title">Mapping Report of gi|549058243|emb|JB409847.1|</h1>
    <h2 class="subtitle">
        Report created with <a href="http://sequana.readthedocs.org">Sequana</a> (0.1.14) and reports (0.2.1).</h2>
    <hr>
</header>

    

    

      

<!--    <div class="contents local topic" id="contents">
        <ul class="simple">
            <li> 1 - <a class="reference internal" href="#Stats" id="id1">
                Stats</a></li>
            <li> 2 - <a class="reference internal" href="#Dependencies"
                id="id11">Dependencies</a> </li>
        </ul>
    </div>
-->
<p>
Sequencing depth and genome coverage report <br>
<b>Chromosome:</b> gi|549058243|emb|JB409847.1| 

</p>

<!--This page informs of the mapping depth coverage. Some regions are detected
    with an unusual depth coverage and are showed below.-->

<div class="section" id="Coverage">
    <h2><a class="toc-backref" href="#id1">Coverage</a></h2>
    <div>
    The graphics shows the coverage as function of the position in the genome.

    </div>
    <img src=images/coverage_coverage.chrom1.png> </>
</div>

<div class="section" id="Submapping list"> 
Interactive plots are available in the following table. Note that the whole
    reference is split into sub-regions (500,000 bp length at max).

    <div class="table_outer"><table border="1" class="dataframe sortable">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a target="_blank" alt=coverage_mapping_0_19795 href="submapping/coverage_mapping_0_19795.chrom1.html">coverage_mapping_0_19795</a></td>
    </tr>
  </tbody>
</table></div>
</div>


<div class="section" id="Basic stats">

    <h2><a class="toc-backref" href="#id1">Basic stats</a></h2>
<p>The following table gives some basic statistics about the genome coverage. 

<div class="table_outer"><table border="1" class="dataframe sortable">
  <thead>
    <tr style="text-align: right;">
      <th>Metric</th>
      <th>0</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>BOC</td>
      <td>0.97</td>
    </tr>
    <tr>
      <td>CV</td>
      <td>0.25</td>
    </tr>
    <tr>
      <td>DOC</td>
      <td>931.31</td>
    </tr>
    <tr>
      <td>MAD</td>
      <td>71.00</td>
    </tr>
    <tr>
      <td>MAD_normed</td>
      <td>0.04</td>
    </tr>
    <tr>
      <td>median</td>
      <td>988.00</td>
    </tr>
    <tr>
      <td>std</td>
      <td>237.15</td>
    </tr>
  </tbody>
</table></div>

Here are some definitions used:
<ul>
<li>BOC: breadth of coverage, that is proportion is percent of the genome
covered by at least one read. Values below 1 indicates the presence of deleted
regions.</li>
<li>CV: the coefficient of variation</li>
<li>DOC: the sequencing depth (Depth of Coverage), that is the avearge of the
genome coverage shown here above</li>
<li>MAD: median of the absolute median deviation median(|X-median(X)|)</li>
<li>std: standard deviation</li>
</ul>
<br>    
</p>
</div>


<div class="section" id="Normalized coverage">
    <h2><a class="toc-backref" href="#id1">Normalized coverage</a></h2>
    Distribution of the normalized coverage with predicted Gaussian. The red line should be followed the trend of the barplot.
    <div>
        <img src=images/coverage_norm_cov_his.chrom1.png> </>
    </div>
</div>

<div class="section" id="Z-Score distribution">
    <h2><a class="toc-backref" href="#id1">Z-Score distribution</a></h2>
    Distribution of the z-score (normalised coverage); You should see a Gaussian distribution centered around 0. The estimated parameters are mu=1.01 and sigma=0.06.
<div>    
    <img src=images/coverage_zscore_hist.chrom1.png> </>
</div>
</div>

<h2><a class="toc-backref" href="#id1">Regions of Interest</a></h2>
Running median is the median computed along the genome using a sliding window.
The following tables give regions of interest detected by sequana. Here is some captions:
<ul>
    <li>mean_cov: the average of coverage</li>
    <li>mean_rm: the average of running median</li>
    <li>mean_zscore: the average of zscore</li>
    <li>max_zscore: the higher zscore contains in the region</li>
</ul>

<div class="section" id="Low coverage region">
    <h3><a class="toc-backref" href="#id1">Low coverage region</a></h3>
    Regions with a z-score lower than -2.00 and at least one base with a z-score lower than -4.00 are detected as low coverage region. Thus, there are 6 low coverage regions
    <div><div class="table_outer"><table border="1" class="dataframe sortable">
  <thead>
    <tr style="text-align: right;">
      <th>chr</th>
      <th>start</th>
      <th>end</th>
      <th>size</th>
      <th>mean_cov</th>
      <th>mean_rm</th>
      <th>mean_zscore</th>
      <th>max_zscore</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>gi|549058243|emb|JB409847.1|</td>
      <td><a target="_blank" href=submapping/coverage_mapping_0_19795.chrom1.html>3553</a></td>
      <td>4252</td>
      <td><p style="background-color:#FFB976">699.0</p></td>
      <td>24.59</td>
      <td>917.63</td>
      <td>-15.44</td>
      <td>-15.87</td>
    </tr>
    <tr>
      <td>gi|549058243|emb|JB409847.1|</td>
      <td><a target="_blank" href=submapping/coverage_mapping_0_19795.chrom1.html>5400</a></td>
      <td>6211</td>
      <td><p style="background-color:#FFC77E">811.0</p></td>
      <td>400.69</td>
      <td>877.59</td>
      <td>-8.68</td>
      <td>-12.67</td>
    </tr>
    <tr>
      <td>gi|549058243|emb|JB409847.1|</td>
      <td><a target="_blank" href=submapping/coverage_mapping_0_19795.chrom1.html>9351</a></td>
      <td>9548</td>
      <td><p style="background-color:#C47C4F">197.0</p></td>
      <td>763.77</td>
      <td>1043.97</td>
      <td>-4.40</td>
      <td>-4.73</td>
    </tr>
    <tr>
      <td>gi|549058243|emb|JB409847.1|</td>
      <td><a target="_blank" href=submapping/coverage_mapping_0_19795.chrom1.html>17045</a></td>
      <td>17048</td>
      <td><p style="background-color:#9E633F">3.0</p></td>
      <td>1.00</td>
      <td>1007.00</td>
      <td>-15.85</td>
      <td>-15.87</td>
    </tr>
    <tr>
      <td>gi|549058243|emb|JB409847.1|</td>
      <td><a target="_blank" href=submapping/coverage_mapping_0_19795.chrom1.html>17052</a></td>
      <td>17053</td>
      <td><p style="background-color:#9E633F">1.0</p></td>
      <td>4.00</td>
      <td>1006.00</td>
      <td>-15.80</td>
      <td>-15.80</td>
    </tr>
    <tr>
      <td>gi|549058243|emb|JB409847.1|</td>
      <td><a target="_blank" href=submapping/coverage_mapping_0_19795.chrom1.html>17224</a></td>
      <td>17523</td>
      <td><p style="background-color:#D88857">299.0</p></td>
      <td>664.20</td>
      <td>986.39</td>
      <td>-5.31</td>
      <td>-6.83</td>
    </tr>
  </tbody>
</table></div> <p>Download the CSV <a href="low_coverage.csv">file</a> </p> </div>
</div>

<div class="section" id="High coverage region">
    <h3><a class="toc-backref" href="#id1">High coverage region</a></h3>
    Regions with a z-score higher than 2.00 and at least one base with a z-score higher than 4.00 are detected as high coverage region. Thus, there are 1 high coverage regions
    <div><div class="table_outer"><table border="1" class="dataframe sortable">
  <thead>
    <tr style="text-align: right;">
      <th>chr</th>
      <th>start</th>
      <th>end</th>
      <th>size</th>
      <th>mean_cov</th>
      <th>mean_rm</th>
      <th>mean_zscore</th>
      <th>max_zscore</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>gi|549058243|emb|JB409847.1|</td>
      <td><a target="_blank" href=submapping/coverage_mapping_0_19795.chrom1.html>4757</a></td>
      <td>5014</td>
      <td><p style="background-color:#FFC77E">257.0</p></td>
      <td>1065.21</td>
      <td>823.0</td>
      <td>4.42</td>
      <td>4.68</td>
    </tr>
  </tbody>
</table></div> <p>Download the CSV <a href="high_coverage.csv">file</a> </p> </div>
</div>









      <hr/>

   
    command used:<br> 
    <pre>
    /home/sequana/miniconda3/bin/sequana_coverage --input JB409847.bed -c 1
    </pre>
    

    <div class="footer">
       <div class="logo">
            <!-- put some images here -->
       </div>

      <div class="copyright">
          <div class="date">Created on 2016-12-06 15:56:47 by sequana</div>
       </div>
   </div>
</div>
</body>
</html>